src/utils/Eip1993Factories.ts

```diff
@@ -11,11 +11,13 @@ import {
   JsonRpcResponse,
   EthersProvider,
 } from '../types/publicTypes';
+import {Provider as ZilliqaProvider} from '@zilliqa-js/core';
```

Comment:
@sudoryan Could we just take or create ZilliqaProvider interface on our side to avoid importing a whole "zilliqa-js" dependency?